__init__.py

```diff
@@ -176,17 +171,8 @@ def update_examples(self):
         """
         Loads or updates skill examples via the skill_info_api.
         """
-        if not self.skill_info_api:
```

Comment:
should this be left as an option in config (if skill is spec'd in config, use the API)? This would keep backwards-compat, but if the Skill API is going to be deprecated, I think date/time in this skill might be the only part left..

Reply:
I think ill leave it as an option, Skill API shouldn't be depreciated its still useful to have

Reply:
fixed in latest commit

Reply:
I'd make one change to prevent using the old default examples skill
```python
# line 75-76
self.skill_info_skill = self.settings.get("examples_skill")  # Drops default skill and requires settings spec
```